spec.html

```diff
@@ -13064,7 +13064,10 @@ <h1>Static Semantics: Early Errors</h1>
       </emu-grammar>
       <ul>
         <li>
-          It is an early Reference Error if IsValidSimpleAssignmentTarget of |LeftHandSideExpression| is *false*.
+          It is an early Reference Error if IsValidSimpleAssignmentTarget of |LeftHandSideExpression| is ~invalid~.
```

Comment:
This still needs to be renamed to not have an “is” prefix` on an enum

Reply:
in the editor's call, we decided to rename this to "AssignmentTargetType". I'll take care of this as part of the rebase and merge.